createdisk.sh

```diff
@@ -2,6 +2,38 @@
 
 set -x
 
+function create_crc_libvirt_sh {
+    hostInfo=$(sudo virsh net-dumpxml ${VM_PREFIX} | grep ${VM_PREFIX}-master-0 | sed "s/^[ \t]*//")
+    masterMac=$(sudo virsh dumpxml ${VM_PREFIX}-master-0 | grep "mac address" | sed "s/^[ \t]*//")
+
+    sed "s|ReplaceMeWithCorrectVmName|${CRC_VM_NAME}|g" crc_libvirt.template > $tarballDirectory/crc_libvirt.sh
+    sed -i "s|ReplaceMeWithCorrectHost|$hostInfo|g" $tarballDirectory/crc_libvirt.sh
+    sed -i "s|ReplaceMeWithCorrectMac|$masterMac|g" $tarballDirectory/crc_libvirt.sh
+
+    chmod +x $tarballDirectory/crc_libvirt.sh
+}
+
+function create_disk_image {
+    sudo cp /var/lib/libvirt/images/${VM_PREFIX}-master-0 $tarballDirectory
+    sudo cp /var/lib/libvirt/images/${VM_PREFIX}-base $tarballDirectory
+
+    sudo chown $USER:$USER -R $tarballDirectory
+    qemu-img rebase -b ${VM_PREFIX}-base $tarballDirectory/${VM_PREFIX}-master-0
+    qemu-img commit $tarballDirectory/${VM_PREFIX}-master-0
+
+    # TMPDIR must point at a directory with as much free space as the size of the
+    # image we want to sparsify
+    TMPDIR=$(pwd)/$tarballDirectory virt-sparsify $tarballDirectory/${VM_PREFIX}-base $tarballDirectory/${CRC_VM_NAME}.qcow2
+    rm -fr $tarballDirectory/.guestfs-*
+
+    rm -fr $tarballDirectory/${VM_PREFIX}-master-0 $tarballDirectory/${VM_PREFIX}-base
+}
+
```

Comment:
Can we add a function here to get all the tools required for this script to complete successfully installed on the system or not like `qemu-img` etc.